config/activity.go

```diff
@@ -0,0 +1,26 @@
+package config
+
+type AllowActivities struct {
+	SyncUser                 Activity `mapstructure:"syncUser" json:"syncUser"`
+	FetchBids                Activity `mapstructure:"fetchBids" json:"fetchBids"`
+	EnrichUserFPD            Activity `mapstructure:"enrichUfpd" json:"enrichUfpd"`
+	ReportAnalytics          Activity `mapstructure:"reportAnalytics" json:"reportAnalytics"`
+	TransmitUserFPD          Activity `mapstructure:"transmitUfpd" json:"transmitUfpd"`
+	TransmitPreciseGeo       Activity `mapstructure:"transmitPreciseGeo" json:"transmitPreciseGeo"`
+	TransmitUniqueRequestIds Activity `mapstructure:"transmitUniqueRequestIds" json:"transmitUniqueRequestIds"`
+}
+
+type Activity struct {
+	Default *bool          `mapstructure:"default" json:"default"`
+	Rules   []ActivityRule `mapstructure:"rules" json:"rules"`
+	Allow   bool           `mapstructure:"allow" json:"allow"`
+}
+
+type ActivityRule struct {
+	Condition ActivityCondition `mapstructure:"condition" json:"condition"`
+}
+
+type ActivityCondition struct {
```

Comment:
I believe these will need to be fleshed out a little further when we actually implement the activity infrastructure.

Reply:
This is everything needed for Phase 3, right? In Phase 4 there is also planned for `privacyreg`, but let's finish out Phase 3 first. The implementation for Phase 3 isn't started yet. To keep PRs small, this is just the config.

Reply:
Phase 3 defines namespace and wildcards, but those are details of the string value parsing and are not represented at this level. Perhaps they could be though? Did you intend to suggest adding a "name scoped value" of some sort with custom unmarshal/marshal methods to handle the parsing?

Reply:
Actually, this is Viper and not standard json. Probably best to leave the "named scope" parsing for other parts of code so we don't have too much Viper specialized functionality.